### PR TITLE
Using Figure captions instead of lists in documentation

### PR DESCRIPTION
**Pull request by smaffulli**
Lists, ordered and unordered, seem to create issues with zendesk
rendering of the output html. Instead of lists, it's preferrable
to use image caption in the .. figure:: directive.

This document is a first attempt at using this tecnique. The output
is also compliant with styleguide where they require the image to be
shown *before* the text describing the actions to take.

### CODE REVIEW

**Comment by justinknierim** [comment]
Yeah, that doesn't look bad.  I still am rather fond of ordered lists, but this looks fine too.

**Comment by squidboylan** [comment]
Yeah i would prefer ordered lists as well, but if we cant get them to work then this should be good enough. The only issue i have with this is that the indentation is 3 spaces instead of the standard 4 for the lines that used to be part of the ordered lists.

**Comment by squidboylan** [comment]
lgtm except the blocks of text are indented 7 spaces now instead of 4 (used to be 3)

**Comment by smaffulli** [comment]
The indentation is to respect the 'caption'  directive: the line with .. image:: is indented 4 spaces in order for the image to fall inside the <li> tag, and the caption below needs to stay inside the .. image:: indentation, 7 or more spaces (I aligned with the 'i' of image).

**Comment by squidboylan** [comment]
if it has to be 7 or more spaces maybe we should stick with multiples of 4 and do 8 spaces? not sure if it matters too much

**Comment by squidboylan** [comment]
I'm ok with it either way

File: source/dreamobjects/tutorials/how-to-use-arq-with-dreamobjects.rst [hunk 1]
```diff
@@ -14,108 +14,144 @@ a password before they're uploaded so your data is protected.
...
 
 Connecting with Arq on Windows
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-1. Launch Arq.  When launching Arq for the first time, you are prompted to
-   choose your backup destination type.  Click the 'Other S3-Compatible
-   Service' option, and then click **continue**.
 
-.. figure:: images/Arq-1-win.png
+1. Lunch Arq:
```
Comment (justinknierim) [inline source/dreamobjects/tutorials/how-to-use-arq-with-dreamobjects.rst]: Lunch Arq!  :)

**Comment by justinknierim** [comment]
I think it looks good.  But sadly we won't know how the ordered lists look, if it works, and if the formatting is acceptable to the knowledge base team until it is published.

**Comment by smaffulli** [comment]
Let's merge and see how it looks on ZD

**Pull request merged**